pkg/server/plugin/upstreamauthority/awspca/pca.go

```diff
@@ -88,6 +92,13 @@ func (m *PCAPlugin) Configure(ctx context.Context, req *spi.ConfigureRequest) (*
 		return nil, err
 	}
 
+	if config.SupplementalBundlePath != "" {
```

Comment:
Maybe it is worth to log the fact that a supplemental bundle will be used?

Reply:
Yes! Updated